draft-irtf-cfrg-opaque.md

```diff
@@ -408,29 +407,29 @@ API and parameters:
 A Memory Hard Function (MHF) is a slow and expensive cryptographic hash function
 with the following API:
 
-- Harden(msg, params): Repeatedly apply a memory-hard function with parameters
-  `params` to strengthen the input `msg` against offline dictionary attacks.
+- Harden(msg): Apply a memory-hard function to strengthen the input `msg`
+  against offline dictionary attacks.
```

Comment:
What do you suggest as parameters? Should we refer to the default parameters of the MHF?